modules/test/playwright/tests/style-book-web/previewStyleBookOnDisplayPageTemplates.spec.ts

```diff
@@ -0,0 +1,221 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {pageEditorPagesTest} from '../../fixtures/pageEditorPagesTest';
+import {styleBookPageTest} from '../../fixtures/styleBookPageTest';
+import {workflowPagesTest} from '../../fixtures/workflowPagesTest';
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import getRandomString from '../../utils/getRandomString';
+import {blogsPagesTest} from '../blogs-web/fixtures/blogsPagesTest';
+
+const test = mergeTests(
+	blogsPagesTest,
+	isolatedSiteTest,
+	loginTest(),
+	pageEditorPagesTest,
+	styleBookPageTest,
+	workflowPagesTest
+);
+
+test('Add a heading fragment and view the display page templates is shown in preview type selector', async ({
+	displayPageTemplatesPage,
+	page,
+	pageEditorPage,
+	site,
+	styleBooksPage,
+}) => {
+	const displayPageTemplateNameBlogsEntry = 'Blogs ' + getRandomString();
+	const displayPageTemplateNameWebContent = 'WebContent' + getRandomString();
```

Comment:
I think having the prefix "blogs" and "WebContent" are unnecessary, you are generating a random string anyway.

Reply:
For instance, you are not adding a prefix for the `styleBookName` variable on line 36